tests/basic/storage_pool_test.go

```diff
@@ -1556,9 +1558,9 @@ func getVolumeWithMinimumSize(contexts []*scheduler.Context, size uint64) (*volu
 				}
 			}
 		}
-		return nil, true, fmt.Errorf("error getting volume with size atleast 10 GiB used")
+		return nil, true, fmt.Errorf(fmt.Sprintf("error getting volume with size atleast %d GiB used", size))
```

Comment:
fmt.Sprintf is not required as you are already formatting using errrorf

Reply:
Got incorporatd as part of your changes when I rebased with Master